Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqDataset.Tests.ps1

```diff
@@ -236,6 +235,46 @@ Describe "Remove-GcbqDataset" {
     }
 }
 
-#TODO(ahandley): Describe "Set-GcbqDataset".
+Describe "Set-GcbqDataset" {
+
+    It "should update trivial metadata fields via pipeline (not DatabaseId)" {
+        try {
+            New-GcbqDataset "test_dataset_id1" -Name "Testdata" -Description "Some interesting data!" -Expiration $oneDaySec
+            $data = Get-GcbqDataset "test_dataset_id1"
+            $data.FriendlyName = "Some Test Data"
+            $data.Description = "A new description!"
+            $data.DefaultTableExpirationMs = $threeDayMs
+            $data | Set-GcbqDataset -outvariable data
+            $data | Should Not BeNullOrEmpty
+            $data.FriendlyName | Should Be "Some Test Data"
+            $data.Description | Should Be "A new description!"
+            $data.DefaultTableExpirationMs | Should Be $threeDayMs
+        }
+        finally {
+            Get-GcbqDataset "test_dataset_id1" | Remove-GcbqDataset
+        }
+    }
+
+    It "should update trivial metadata fields via parameter (not DatabaseId)" {
+        try {
+            New-GcbqDataset "test_dataset_id1" -Name "Testdata" -Description "Some interesting data!" -Expiration $oneDaySec
+            $data = Get-GcbqDataset "test_dataset_id1"
+            $data.FriendlyName = "Some Test Data"
+            $data.Description = "A new description!"
+            $data.DefaultTableExpirationMs = $threeDayMs
+            $data = Set-GcbqDataset -ByObject $data
```

Comment:
I would add a blank line above here to help show visually this is the line being tested.